pprofile.py

```diff
@@ -883,14 +962,37 @@ def runpath(path, argv, filename=None, threads=True, verbose=False):
     """
     _run(threads, verbose, 'runpath', filename, path, argv)
 
-_allsep = os.sep + (os.altsep or '')
+def _qCacheGringAdapt(path):
+    '''
+    qCacheGrind (windows build) needs at least one UNIX separator
+    in path to find the file. Adapt here even if this is probably
+    more of a qCacheGrind issue...
+    '''
+
+    if os.path.sep != "/":
```

Comment:
In the case of `qcachegrind`, it could be made into a local function.  But if you don't like it to be a function at all, then it can just go back to how it was.  I guess I liked it as a function so the reader only had to read one (hopefully) descriptive function-call line - not that it was too long to start with.